client/config/crd/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -92,3 +92,77 @@ spec:
     served: true
     storage: true
```

Comment:
v1beta1 should have storage: false.

Reply:
same as https://github.com/kubernetes-csi/external-snapshotter/pull/1312#discussion_r2187275506